contributors/design-proposals/propagation.md

```diff
@@ -89,18 +100,23 @@ The new `HostPathVolumeSource` will look like:
 
 ```go
 const (
-	PropagationShared  PropagationMode = "Shared"
-	PropagationSlave   PropagationMode = "Slave"
-	PropagationPrivate PropagationMode = "Private"
+	MountPropagationRShared  MountPropagationMode = "RShared"
+	MountPropagationRSlave   MountPropagationMode = "RSlave"
+	MountPropagationPrivate  MountPropagationMode = "Private"
 )
 
 type HostPathVolumeSource struct {
 	Path string `json:"path"`
 	// Mount the host path with propagation mode specified. Docker only.
-	Propagation PropagationMode `json:"propagation,omitempty"`
+	MountPropagation MountPropagationMode `json:"propagation,omitempty"`
 }
 ```
 
+The default mount propagation is `rslave`. Any HostPath can ask for `private`.
```

Comment:
Why would anyone ask for private?  What value does the extra surface area provide?